test/spec/GarnSpec.hs

```diff
@@ -123,7 +123,16 @@ spec = do
                 `shouldBe` unindent
                   [i|
                     [garn] Error: Version mismatch detected:
-                    'garn' (the cli tool) is not compatible with the version of the garn typescript library you're using.
-                    Try installing version `test version string` of 'garn' (the cli tool).
-                    (Internal details: Error in $: key \"tag\" not found)
+                      garn cli tool version: v0.0.15
+                      garn typescript library version: <testTsLibVersion>
+
+                      Either:
+
+                        Install version <testTsLibVersion> of the garn cli tool.
+                        See https://garn.io/docs/getting_started#updating-garn for how to update.
+
+                        Or: Use version v0.0.15 of the typescript library.
```

Comment:
A bit weird formatting. Maybe move the `Or` to a separate line like the `Either`, or have it be a list? E.g.:

```
Either:

1) Install version <testTsLibVersion> of the garn cli tool.
    (See https://garn.io/docs/getting_started#updating-garn for how to update.) Or;
2) Use version v0.0.15 of the typescript library.
```

Reply:
Done in 659bf17e.